### DISCUSSION TITLE
Welcome to django-graphbox Discussions!

### DISCUSSION THREAD

**Discussion by yefeza** [yefeza/django-graphbox#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Yugthakkar04** [comment]
Hello Everyone, 
Yug this side i would like to join this community showing some interest in django framework which is especially a vast source in python and i would like to work as maintaining and security of the adapted programme.

**Comment by Yugthakkar04** [comment]
import asyncio
from tartiflette import Engine, Resolver
@Resolver("Query.hello")
async def resolver_hello(parent, args, ctx, info):
    return "hello " + args["name"]
async def run():
    tftt_engine = Engine("""
    type Query {
        hello(name: String): String
    }
    """)
    result = await tftt_engine.execute(
        query='query { hello(name: "Chuck") }'
    )
    print(result)
    # {'data': {'hello': 'hello Chuck'}}
if __name__ == "__main__":
    loop = asyncio.get_event_loop()
    loop.run_until_complete(run())

this would help as a example for working of django.
